source/partials/_signup-cta.haml

```diff
@@ -0,0 +1,6 @@
+- cta_label ||= 'Sign Up'
+- cta_placeholder ||= 'Email'
+%a.btn.btn-primary{ href: 'https://dashboard.aptible.com/signup' }= cta_label
+-# %form.signup-cta.single-line{ action: 'http://localhost:4200/signup' }
+-#   %input.single-line__input{ type: 'email', placeholder: cta_placeholder }
+-#   %button.single-line__submit{ type: 'submit' }= cta_label
```

Comment:
This is looking great. If we can reliably pass email via Segment, that'll be great... we ultimately will want to customize the /signup page based on who the person is (and Segment sends all that info). Sending email to /signup via Segment, rather than querystring, gets us a head start there.